src/Adapter/MSTest.CoreAdapter/Extensions/MethodInfoExtensions.cs

```diff
@@ -143,6 +145,14 @@ internal static string GetAsyncTypeName(this MethodInfo method)
         /// </param>
         internal static void InvokeAsSynchronousTask(this MethodInfo methodInfo, object classInstance, params object[] parameters)
         {
+            if (methodInfo.GetParameters() != null && parameters == null)
```

Comment:
Please explain the parameters null check

Reply:
Added comments.